server/app/mutations/stacks/patch.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'common'
+
+module Stacks
+  class Patch < Mutations::Command
+    include Common
+
+    required do
+      model :stack_instance, class: Stack
+    end
+
+    optional do
+      model :grid, class: Grid
+      array :labels
+    end
+
+    def validate
```

Comment:
the `array` clause already validates that it is an array.

Looks like this can be done:

```ruby
array :labels do
  string
end
```

it works a bit strangely though:

```
task.new(labels: [1,2,3])
=>
:labels => ["1", "2", "3"]
```

but catches this:

```
task.new(labels: [nil])
=> 
{"labels" => ["Labels[0] can't be nil"]}
```

Reply:
@kke thanks for the feedback - adjusted the code for that + test coverage

Reply:
✅ RESOLVED